### PR TITLE
docs(joy_controller): joystick keymap fix in readme

### PR DESCRIPTION
**Pull request by sfukuta**
## Description

Convert the joystick key map in the readme from P65 to DS4.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: control/joy_controller/README.md [hunk 1]
```diff
@@ -42,23 +42,23 @@
 | `max_backward_velocity`   | double | absolute max velocity to go backward                                                                               |
 | `backward_accel_ratio`    | double | ratio to calculate deceleration (commanded acceleration is -ratio \* operation)                                    |
 
-## P65 Joystick Key Map
```
Comment (kenji-miyake) [inline control/joy_controller/README.md:45]: Please do not remove `P65` and newly add `DS4`.

**Review by kenji-miyake — CHANGES_REQUESTED** [review]
I'll change the title from `fix` to `docs` because this PR won't change any behavior.

**Comment by kenji-miyake** [comment]
Also, please read the guidelines and check off the checkboxes.
image

**Comment by kenji-miyake** [comment]
Also, the DCO check has failed.
image

Please confirm the [CI checks](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/#ci-checks) section and fix the error.

File: control/joy_controller/README.md (hunk 1 above)
Comment (sfukuta) [reply control/joy_controller/README.md]: Done

**Comment by sfukuta** [comment]
DCO check has successful

File: control/joy_controller/README.md [hunk 2]
```diff
@@ -62,3 +62,24 @@
 | Autoware Disengage   | Y                     |
 | Vehicle Engage       | PS                    |
 | Vehicle Disengage    | Right Trigger         |
+
+## DS4 Joystick Key Map
+
+| Acceleration         | R2                                       |
+| -------------------- | ---------------------------------------- |
+| Brake                | □ or Right Stick Up Down or Left Trigger |
```
Comment (kenji-miyake) [inline control/joy_controller/README.md:70]: Maybe this is correct? :thinking: 

```suggestion
| Action               | Button                                   |
| -------------------- | ---------------------------------------- |
| Acceleration         | R2, ×, or Right Stick Up                 |
| Brake                | L2, □, or Right Stick Down               |
```

Comment (kenji-miyake) [reply control/joy_controller/README.md:70]: Could you also fix the table header of the P65's keymap?
image

Comment (sfukuta) [reply control/joy_controller/README.md]: Done

**Review by kenji-miyake — APPROVED** [review]
Thank you!

**Comment by sfukuta** [comment]
@kenji-miyake Please workflow review and run.
workflow

**Comment by sfukuta** [comment]
Thank you!

**Pull request merged**